predicates/src/test/scala/com/wheaties/predicate/PredicateSpec.scala

```diff
@@ -61,6 +61,24 @@ class PredicateSpec extends WordSpecLike
       val filtered = sampleList.filter(Modulo(2, 0) or (Modulo(3, 0) and LessThen(6)))
       filtered should equal(List(2, 3, 4, 6))
     }
+
+    "have implicit `not` method" in new SpecExamples {
+//      val filtered = sampleList.filter(Predicate1.not.not(Modulo(2, 0)))
+      val filtered = sampleList.filter(Modulo(2, 0).not)
+      filtered should equal(List(3, 5))
+    }
+
+    "have accompanying Always" in new SpecExamples {
+//      val filtered = sampleList.filter(Modulo(2, 0) or Always1)
```

Comment:
Commented out line worked with old API. Such API is impossible to achieve using just macro annotation because it may return just annotated type and its companion object (no other top level objects). So instead I create inner Object which is then available via field `Predicate1.always`